tests/utils/test_templates.py

```
@@ -0,0 +1,538 @@
+
+        # Should have directional query format
+        assert "alice's understanding of bob" in result.lower()
+        assert (
+            "<working_representation>Alice thinks Bob is helpful</working_representation>"
+            in result
+        )
+        assert (
+            "The user's known biographical information:\n" "Name: Alice\n"
+        ) in result
+        assert (
+            "The target's known biographical information:\n" "Name: Bob\n"
```

Ruff (0.14.3)

Implicitly concatenated string literals on one line

Combine string literals

(ISC001)